bundle-eoan-train.yaml

```diff
@@ -134,7 +134,7 @@ services:
     num_units: 3
     options:
       expected-osd-count: 3
-      source: distro
+      source: cloud:bionic-stein
```

Comment:
OK so this just won't work - the UCA does not have context outside of LTS releases so this will generate errors on install.

This probably means with don't have an OpenStack on LXD solution for Eoan right now which is not ideal.

Reply:
This should be solved by the series addition on the line below